cybersyn/scripts/layout.lua

```diff
@@ -97,7 +97,13 @@ function set_train_layout(map_data, train)
 			item_slot_capacity = item_slot_capacity + #inv
 		elseif carriage.type == "fluid-wagon" then
 			layout[#layout + 1] = 2
-			fluid_capacity = fluid_capacity + carriage.prototype.fluid_capacity
+
+			--This is absolutely the only way of knowing how big a fluid wagon is with quality right now
```

Comment:
This could be fixed by asking for `LuaEntityPrototype::get_fluid_capacity(quality?)`

Reply:
Unfortunately, due to what looks like an oversight, that API doesn't exist. There is a corresponding `get_inventory_size(quality)` but no `get_fluid_capacity`